nodes/scene/get_objects_data.py

```diff
@@ -296,24 +317,24 @@ def process(self):
                     # from 3dview while in edit mode when using obj.to_mesh.
                     me = obj.data
                     bm = bmesh.from_edit_mesh(me)
-                    vers, edgs, pols = pydata_from_bmesh(bm)
+                    # verts, edgs, pols = pydata_from_bmesh(bm)
 
                     if o_vs:
-                        vs.append(vers)
+                        verts = [ Vector(v.co) for v in bm.verts]  # v.co is a Vector()
```

Comment:
my only surprise here is that `verts` will be holding a list of Vectors -- obviously in anticipation for the 'if self.apply_matrix` being true.  from what I can tell .. if apply_matrix is false, then the node will be outputting lists of Vector lists into the node tree.

Reply:
> Hi @zeffii . I puzzle out now! It is convenient to rename a file node after move it to old/nodes! )))

much better !

Reply:
>> if apply_matrix is false, then the node will be outputting lists of Vector lists into the node tree
Hi @zeffii . I agree with you. Fix it.

Reply:
> if apply_matrix is false, then the node will be outputting lists of Vector lists into the node tree
Hi @zeffii. I agree with you. I fixed it.